srcpkgs/retroarch/template

```diff
@@ -29,12 +29,11 @@ build_options="ffmpeg opengl jack pulseaudio sdl2 x11 vulkan qt5"
 build_options_default="ffmpeg"
 
 case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*|ppc64*) build_options_default+=" opengl pulseaudio sdl2 x11 vulkan";;
-	armv[67]*) broken="https://build.voidlinux.org/builders/armv7l_builder/builds/19214/steps/shell_3/logs/stdio"
+	i686*|x86_64*|ppc64*|arm[67]) build_options_default+=" opengl pulseaudio sdl2 x11 vulkan";;
```

Comment:
what's the reason for the change in this line? We don't have architectures that are called armv6 or armv7.

Reply:
Will fix